tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+- name: Couchbase | Install Couchbase Repo
+  yum:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - http://packages.couchbase.com/releases/couchbase-release/couchbase-release-1.0-0-x86_64.rpm
+    - couchbase-server-4.5.0
+
+- name: Couchbase | Create Data Directory
+  file:
+    path: /data
```

Comment:
Make this configurable (but default to /data; or potentially the /var/lib/couchbase that th package defaults to?)

Reply:
I did actually have a data path but haven't used the var here - will amend!